src/__internal__/utils/helpers/tags/tags-specs/index.ts

```diff
@@ -1 +1,2 @@
-export { elementsTagTest, rootTagTest } from "./tags-specs";
+// eslint-disable-next-line import/prefer-default-export
+export { rootTagTest } from "./tags-specs";
```

Comment:
**suggestion**: To avoid this you could export the `rootTagTest` as default in `tags-specs.ts`